docs/reference/tags/variant_pricing/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Variant Pricing
+parent: Tags
+grand_parent: Reference
+---
+
+# Variant Pricing
+
+The `variant_pricing` tag renders a self-updating `<span>` which can
+interactively fetch and display up-to-date pricing information for a given
+variant, provided an adult count/start date/end date combination.
```

Comment:
Are these variables all required, or are some optional? I believe it also accepts other variables, can these please be listed (along with whether they are required or not)?

Reply:
Good point, will indicate more clearly which variables are required and not.

The current package step _is_ a parameter, but you can't specify it explicitly, we infer it from the page we're rendering it on.

Reply:
Addressed 6861ff5